src/Kunstmaan/AdminBundle/Resources/views/authentication/login.html.twig

```diff
@@ -14,43 +17,46 @@
                 {{ 'security.login.username'|trans }}
             </label>
-            <input type="text" value="{{ last_username }}" name="_username" class="form-control form-group--icon-in-control__form-control" placeholder="{{ 'security.login.username'|trans }}" autocomplete="username" required autofocus>
-            <i class="fa fa-user form-group--icon-in-control__icon"></i>
+            <input class="form-control" id="username" type="text" name="_username" value="{{ last_username }}" placeholder="{{ 'security.login.username'|trans }}" autocomplete="username" required autofocus>
         </div>
 
-        <div class="form-group form-group--icon-in-control">
-            <label for="_password" class="label--lighter">
+        <div class="form-group">
+            <label for="password" class="form-control-label">
                 {{ 'security.login.password'|trans }}
             </label>
-            <input type="password" name="_password" class="form-control form-group--icon-in-control__form-control" placeholder="{{ 'security.login.password'|trans }}" autocomplete="current-password" required>
-            <i class="fa fa-key form-group--icon-in-control__icon"></i>
+            <input class="form-control" id="password" type="password" name="_password" value="" placeholder="{{ 'security.login.password'|trans }}" autocomplete="current-password" required>
         </div>
 
-        <div class="form-group login__form__remember-me">
-            <label class="checkbox login__form__remember-me__check">
-                <input type="checkbox" id="remember_me" name="_remember_me" checked>
-                {{ 'security.login.remember_me' | trans }}
-            </label>
+        <div class="app-login__form__alt">
+            <div class="app-login__form__alt__remember-me">
+                <label class="checkbox login__form__remember-me__check">
+                    <input type="checkbox" id="remember_me" name="_remember_me" checked>
+                    {{ 'security.login.remember_me' | trans }}
+                </label>
+            </div>
+            <div class="app-login__form__alt__forgot-pw">
+                <a href="{{ path('kunstmaan_admin_reset_password') }}" class="login__form__forgot-pw__link">
+                    {{ 'kuma_admin.login.forgot_password'|trans }}
+                </a>
+            </div>
         </div>
-
+                
         <input type="hidden" name="_csrf_token" value="{{ csrf_token('authenticate') }}">
 
-        <button type="submit" id="_submit" name="_submit" class="btn btn-brand btn-block btn--raise-on-hover">
-            {{ 'security.login.submit'|trans }}
-        </button>
+        <div class="form-group app-login__form__submit-holder">
+            <button type="submit" id="_submit" name="_submit" class="btn btn--md btn--inverted">
+                {{ 'security.login.submit'|trans }}
+            </button>
+        </div>
     </form>
 
     {% if google_signin_enabled() %}
```

Comment:
Can you check if the signin with google button also show correct in the new layout? (Can be temporary "faked" by commenting the if statement)